src/components/pnp/isGoalStepPlannedInsideProject.test.ts

```diff
@@ -0,0 +1,87 @@
+import { jest } from '@jest/globals';
+import { promises as fs } from 'fs';
+import { DateTime } from 'luxon';
+import { CalendarDate, DateInterval } from '../../common';
+import * as FiscalYear from '../../common/fiscal-year';
+import { WorkBook } from '../../common/xlsx.util';
+import { Pnp } from '../../components/pnp';
+import {
+  isGoalStepPlannedInsideProject,
+  stepPlanCompleteDate,
+} from './isGoalStepPlannedInsideProject';
+import { PlanningSheet } from './planning-sheet';
+
+const fiscalYear2019 = DateInterval.fromDateTimes(
+  CalendarDate.local(2018, 10, 1),
+  CalendarDate.local(2019, 9, 30)
+);
+
+const mockFullFiscalYear = jest.spyOn(FiscalYear, 'fullFiscalYear');
+mockFullFiscalYear.mockReturnValue(fiscalYear2019);
+
+//test cases for stepPlanCompleteDate
+describe('stepPlanCompleteDate', () => {
+  let planningSheet: PlanningSheet;
+
+  beforeAll(async () => {
```

Comment:
You can consolidate the two `beforeAll` statements into one and put it at the top of the file before the first `describe`.

Reply:
updated